### DISCUSSION TITLE
JSBSim Movement Component rotation issue in UE5

### DISCUSSION THREAD

**Discussion by ugenethehobo** [JSBSim-Team/jsbsim#1207] [Q&A] [answered]
Hello,
I have my model set up correctly in UE5, and have a working JSBSim Movement component. However, the component is rotated 90 degrees and cannot be changed regardless of what I try. This is the same for every aircraft model I choose, so in effect, I fly sideways in game. Any help would be immensely helpful!
![MiG-23Error](https://github.com/user-attachments/assets/beaff252-d639-4561-827f-9ba4de19d141)

**Comment by bcoconni** [comment]
@gallonmate, any idea about what's going wrong here ?

**Comment by gallonmate** [comment, marked as answer]
@ugenethehobo You need to study the reference project and example blueprint. The sample airplane has a root component placed before the skeletal mesh. This allows for the skeletalmesh to be rotated 90 degrees in order to match JSBSim. You can add a scene component to be used as the root component.
![image](https://github.com/user-attachments/assets/51d24227-875a-40ee-8cbd-51be34728ce7)

**Reply by ugenethehobo** [reply]
The sample project refuses to open, and building in VS crashes on build. So this is perfect, thank you!
